### ISSUE TITLE
Conversion of pretrained TensorFlow model's logits layer into pytorch?

### ISSUE THREAD

**Issue by kouximei** [timesler/facenet-pytorch#106]
First of all, this is a fantastic project and I really really appreciate your help! Currently, I'm trying to re-run the conversion of TensorFlow model into Pytorch model, but I encounter some issues regarding the 'logits' layer. First I cloned this repo with submodules. Then I downloaded the pertained models [20180402-114759](https://drive.google.com/file/d/1EXPBSXwTaqrSC0OhUdXNmKSh9qJUQ55-/view) and [20180408-102900](https://drive.google.com/file/d/1R77HmFADxe87GmoLwzfgMu_HY0IhcyBz/view) from [davidsandberg's repo](https://github.com/davidsandberg/facenet) and put them under the /data folder. But when I run the tensorflow2pytorch() function, the following error occurs:

<img width="949" alt="image" src="https://user-images.githubusercontent.com/69120110/89150762-ec680380-d524-11ea-899e-a31cb1c205e1.png">

I think the reason is that the default constructer of InceptionResnetV1 sets "Classify=False", which prevents the forward function from creating the logits layer ([inception_resnet_v1.py](https://github.com/timesler/facenet-pytorch/blob/38717331d19b2fa51a1c3111c66e9b7067f2dcf8/models/inception_resnet_v1.py) line 299). So I modified [tensorflow2pytorch.py](https://github.com/timesler/facenet-pytorch/blob/38717331d19b2fa51a1c3111c66e9b7067f2dcf8/models/utils/tensorflow2pytorch.py) line 317  to include the logits layer and changed it into this:
`mdl = InceptionResnetV1(num_classes=8631, classify=True).eval()`
Am I doing the correct thing? Or is there something else that I missed?

After that, in order to examine the outputs, I also changed the compare_model_outputs() function in tensorflow2pytorch.py to make both the TensorFlow model and the Pytorch model to return the embeddings, which turn out to be identical:

<img width="965" alt="image" src="https://user-images.githubusercontent.com/69120110/89151991-c6902e00-d527-11ea-9958-d0a52fd54bac.png">

However, the embeddings are returned before the logits layer, which means I still cannot test whether the classification outputs of the logits layer is correct. More specifically, I couldn't figure out how to let the TensorFlow model return the classification results. Would it be possible for me to get the classification probs in a way similar to what you did in [tensorflow2pytorch.py](https://github.com/timesler/facenet-pytorch/blob/38717331d19b2fa51a1c3111c66e9b7067f2dcf8/models/utils/tensorflow2pytorch.py) line 232

> embeddings = tf.get_default_graph().get_tensor_by_name("embeddings:0")?

Thanks in advance!
